superset/dashboards/api.py

```diff
@@ -205,6 +206,7 @@ def __init__(self) -> None:
     @protect()
     @safe
     @statsd_metrics
+    @event_logger.log_this
```

Comment:
`log_this` depends on the DB but runs before safe, so if something fails a non handled crash will surface to the user